### DISCUSSION TITLE
H1 2021 Roadmap

### DISCUSSION THREAD

**Discussion by arnaudgiuliani** [InsertKoinIO/koin#1015] [General]
Hello,

just a few words to give an overview of the incoming milestones for Koin:

- 3.0.1 - The next Koin Multiplatform version, bringing new internal architecture & new modules follow-up like Jetpack compose. The version is almost ready, but we need to adjust the native part with some tests.
- 2.3.0 (Previously seen as 2.2.3 fix version). This version target to come back over breakings from last 2.2.2. We'll bring better Scope AP and ViewModel API. jetpack compose will be kept only under 3.0.1 and further branch, allowing to easily follow intermediate versions.

Regarding the binary publication process, we are currently moving to maven central to keep things continued. The impact for you will be the renaming of maven group info. We will push a new blog post when it will be ready: https://insert-koin.io/blog

We will surely ask you some feedbacks about those incomings version 🙂
Many thanks 🙏

**Comment by BulatMukhutdinov** [comment]
Hi!
Is there a sample project using Koin 3.0.1  with Kotlin mpp?
Currently, I stack with this issue https://github.com/InsertKoinIO/koin/issues/997

**Reply by arnaudgiuliani** [reply]
seems fixed then
